### DISCUSSION TITLE
Heatmap

### DISCUSSION THREAD

**Discussion by jllucero** [opengeos/leafmap#63] [Q&A] [answered]
Hello.
I'm trying to add a heatmap layer to my map.  However, when I run the code  "Map.add_layer(heatmap)"  I get the error

AttributeError: 'Heatmap' object has no attribute 'add_to'

Any suggestions?
Thank you.

**Comment by giswqs** [comment]
See https://stackoverflow.com/a/54756617

**Reply by jllucero** [reply]
It works!!!!  Thank you so much!  This looks great.

**Comment by giswqs** [comment, marked as answer]
You can now create heat map from csv. See notebook example: https://leafmap.org/notebooks/24_heatmap

![](https://i.imgur.com/LqcRZnK.gif)
